python/paddle/autograd/backward_mode.py

```diff
@@ -80,7 +92,9 @@ def backward(tensors, grad_tensors=None, retain_graph=False):
 
     """
 
-    def check_tensors(in_out_list, name):
+    def check_tensors(
+        in_out_list: list[Tensor] | tuple[Tensor, ...] | Tensor, name: str
+    ) -> list[Tensor] | tuple[Tensor, ...]:
```

Comment:
```suggestion
    def check_tensors(
        in_out_list: Sequence[Tensor] | Tensor, name: str
    ) -> Sequence[Tensor]:
```

另外，104、109、126 行的 `isinstance(x, (paddle.Tensor, core.eager.Tensor))` 属于历史遗留写法，现在 `paddle.Tensor` 和 `core.eager.Tensor` 是同一个东西，可以合并为 `isinstance(x, paddle.Tensor)`

Reply:
上述内容已修正，并对代码部分做了更新，谢谢您的suggestion！